processor/src/main/java/org/mapstruct/ap/internal/model/source/Mapping.java

```diff
@@ -556,20 +545,19 @@ public Mapping popSourceReference() {
         return null;
     }
 
-    public List<String> getDependsOn() {
-        return dependsOn;
+    /**
+     *  mapping can only be reversed if the source was not a constant nor an expression nor a nested property
+     *  and the mapping is not a 'target-source-ignore' mapping
+     *
+     * @return true when the above applies
+     */
+    public boolean canInverse() {
```

Comment:
Maybe we should name the method `canReverse`? The Javadoc already uses the `reverse` verb to explain what it means 😄.

Reply:
I had long discussions on terminology in the past.. we settled on `inverse` lets make consistent with annotation..

Reply:
I skipped the unit test from this action.. source code is consistent now.